config/development.json

```diff
@@ -165,6 +165,7 @@
 		"upgrades/wpcom-monthly-plans": true,
 		"upsell/concierge-session": true,
 		"woocommerce/extension-referrers": true,
+		"woop": true,
```

Comment:
Is this config flag actually used anywhere yet?

Reply:
Nope but I want this in development.json as I have a few things in draft status that use it and we will need flagging for launch to deal with testing plan level stuff.